Example/tvOSSample/tvOSSample/DatabaseViewController.swift

```diff
@@ -0,0 +1,79 @@
+// Copyright 2017 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import FirebaseDatabase
+
+/// A class to demonstrate the power of the Firebase Realtime Database. This will show a number read
```

Comment:
Nit: would prefer if this just read "A class to demonstrate the Firebase Realtime Database API."

Reply:
Done.

Reply:
Done.